css-pseudo-classes/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>css-pseudo-classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2>Hover Button</h2>
+    <form>
+      <button class="Hover" type="button">I'm a button!</button>
```

Comment:
Class names should always be camelCased or kebab-cased. It shouldn't start with a capital letter